assets/src/blocks/Covers/TakeActionCovers.js

```diff
@@ -36,30 +36,39 @@ export const TakeActionCovers = ({ initialRowsLimit, covers, row, loadMoreCovers
           }
 
           return (
-            <div key={title} className='col-lg-4 col-md-6 cover-card-column'>
-              <div className='cover-card card-one' style={{ backgroundImage: `url(${image})` }}>
+            <div key={title} className='col-lg-4 col-md-6'>
+              <div className='cover-card-new'>
                 <a
                   className='cover-card-overlay'
                   data-ga-category='Take Action Covers'
-                  data-ga-action='Image'
+                  data-ga-action='Card'
                   data-ga-label='n/a'
                   href={button_link}
                   aria-label={__('Take action cover, link to ' + title, 'planet4-blocks')}
                 />
+                <a
+                  data-ga-category='Take Action Covers'
+                  data-ga-action='Image'
+                  data-ga-label='n/a'
+                  href={button_link}
+                  aria-label={__('Take action cover, link to ' + title, 'planet4-blocks')}
+                >
+                  <img src={image} />
+                </a>
                 <div className='cover-card-content'>
-                  {tags && tags.map(tag => (
+                  {/* Regardless of how many tags there are, we only show the first one */}
+                  {tags && tags.length > 0 && (
```

Comment:
`> 0` is not necessary in this case. `tags.length` is ok.

Reply:
I thought about that, but then the block shows `0` in place of the tag :/

<img width="756" alt="Screenshot 2021-08-24 at 09 50 45" src="https://user-images.githubusercontent.com/6949075/130578308-7d99f87e-3613-428f-ad42-4dd597befacb.png">

Reply:
This is because you are using the logical operator (`&&`) and returns true|1 or false|0`. When you have an empty Array it will return `0`.
Try replacing to 
```
{tags && tags.length > 0 ? (your code) : null}
```

Reply:
Updated comment.

Reply:
Hmmm yeah I see, not sure that's better than using `> 0` though? It's longer that way and I find it less clear 🤔

Reply:
Yep, I agree that it's more clear using `> 0` but it's really the same!